doc/operator-notes.md

```diff
@@ -31,6 +31,10 @@ If `wipeFilesystem` is set to true, Ignition will always wipe any preexisting fi
 
 If `wipeFilesystem` is set to false, Ignition will then attempt to reuse the existing filesystem. If the filesystem is of the correct type, has a matching label, and has a matching UUID, then Ignition will reuse the filesystem. If the label or UUID is not set in the Ignition config, they don't need to match for Ignition to reuse the filesystem. Any preexisting data will be left on the device and will be available to the installation. If the preexisting filesystem is *not* of the correct type, then Ignition will fail, and the machine will fail to boot.
 
+## Path Traversal and Following symlnks
+
+When resolving paths, Ignition follows symlinks on all but the last element of a path. This ensures symlinks already existing on a filesystem can be overwritten while still following symlinks as expected. When writing files, links, or directories, Ignition does not allow following symlinks outside the specified filesystem. When writing a file, link, or directory, on the `root` filesystem Ignition will follow symlinks as if it were executing in that root; a symlink to `/etc` will be followed to `/etc` on the `root` filesystem. When writing a file, link, or directory to any other filesystem, Ignition will fail if it tries to follow a symlink outside that filesystem.
```

Comment:
commas: `When writing a file, link, or directory on the `root` filesystem,`

Mix of present and future tense.

Reply:
I think the tense mixing makes sense. It's both describing what Ignition will do and the rules which Ignition follows. I think it would definitely be awkward to do it all in the future tense (`Ignition will disallow` sounds weird) and it's easier to understand the cause/effect if the future tense is used for other sections.